snorkel/labeling/model/label_model.py

```diff
@@ -810,7 +814,35 @@ def fit(
         class_balance
             Each class's percentage of the population, by default None
         **kwargs
-            Arguments for changing train config defaults
+            Arguments for changing train config defaults.
+
+            n_epochs
```

Comment:
Let's point to a page in the documentation (https://snorkel.readthedocs.io/) instead. In case these options update, we don't want to have to remember to change them in two places.

Reply:
Hi, I couldn't find this config anywhere in the snorkel docs that's why I added it here. Could you point me to the page where it is?

Reply:
@henryre

Reply:
Great point, it looks like the config isn't included in documentation. This looks like a good place to add it!